perses/annihilation/new_relative.py

```diff
@@ -74,10 +81,22 @@ def __init__(self, topology_proposal, current_positions, new_positions, use_disp
         angle_softening_constant : float
             For bonds between unique atoms and unique-core atoms, soften the force constant at the "dummy" endpoint by this factor.
             If 1.0, do not soften
+        neglected_new_angle_terms : list
+            list of indices from the HarmonicAngleForce of the new_system for which the geometry engine neglected.
+            Hence, these angles must be alchemically grown in for the unique new atoms (forward lambda protocol)
+        neglected_old_angle_terms : list
+            list of indices from the HarmonicAngleForce of the old_system for which the geometry engine neglected.
+            Hence, these angles must be alchemically deleted for the unique old atoms (reverse lambda protocol)
+        forward_1_4_proposal_terms : list
```

Comment:
`forward_1_4_proposal_terms` and `reverse_1_4_proposal_terms` don't seem to be used anywhere in this script

Reply:
i had them as arguments to the `HybridTopologyFactory` in a revious commit but ive since removed them and forgot to remove them from docstrings.  I'll clean it up in the morning